eras/shelley/impl/src/Cardano/Ledger/Shelley/PParams.hs

```diff
@@ -256,8 +237,9 @@ instance Era era => DecCBOR (ShelleyPParams Identity era) where
         <*> decCBOR -- sppMinUTxOValue    :: Natural
         <*> decCBOR -- sppMinPoolCost     :: Natural
 
-instance Era era => ToCBOR (ShelleyPParams Identity era) where
-  toCBOR = toEraCBOR @era
+instance EraPParams era => ToCBOR (ShelleyPParams Identity era) where
+  toCBOR = toEraCBOR @era -- should we invoke the instance for PParams instead? with: toCBOR . PParams
```

Comment:
It seems wrong to duplicate this implementation with the one in core. 
Is it better to reuse it 
```toCBOR = toCBOR . PParams```
and accept the constraint `PParamsHKD Identity era ~ ShelleyPParams Identity era` ?

Reply:
Same here. It can be removed